test/rule_test.dart

```diff
@@ -204,57 +197,10 @@ defineRuleUnitTests() {
   });
 }
 
-/// Test framework sanity
+// Test framework sanity.
```

Comment:
We use a sorter on our code, but the sorter doesn't always preserve the locations of non-doc comments. Please either restore the slash or move the comments inside the method so that they won't be moved away from the method unintentionally (here and elsewhere).

Reply:
Oh interesting. Done!